### ISSUE TITLE
OTR GO:1990209 negative regulation by symbiont of RNA levels in host

### ISSUE THREAD

**Issue by LornaMin** [geneontology/go-ontology#12368]
To whom it may concern,

I am a student in the CACAO class at Texas A&M University. I was looking for a term in which a symbiont negatively regulates host transcription, and I found the following two GO terms:

GO:0052026 modulation by symbiont of host transcription - any process in which an organism modulates the frequency, rate or extent of its host's transcription. The host is defined as the larger of the organisms involved in a symbiotic interaction.

GO:1990209 negative regulation by symbiont of RNA levels in host - any process in which an organism stops, prevents, or reduces the frequency, rate or extent of the RNA levels in the host organism. The host is defined as the larger of the organisms involved in a symbiotic interaction

Given the existence of the term "modulation by symbiont of host transcription," I don't quite understand how "negative regulation by symbiont of RNA levels in host" fits in. The latter term seems to be somewhat vague - based on my knowledge, a lot of processes could cause an organism to stop/prevent/reduce the frequency/rate/extent of RNA levels, including negative regulation of transcription.

I talked to Dr. Jim Hu and Suzi Aleksander about this, and they thought that the latter GO term sounds more like a phenotype or an artefact of an assay than a biological process or function.

I am therefore interested in hearing your opinion on this GO term.

Thanks,
Lorna

**Comment by mcourtot** [comment]
Hi Lorna,

Thank you for the report. I had a look at http://www.ncbi.nlm.nih.gov/pubmed/18703740, the PMID associated with the term GO:1990209 negative regulation by symbiont of RNA levels in host, and the abstract reads "we have identified P. syringae effectors that suppress transcriptional activation of some PAMP-responsive miRNAs or miRNA biogenesis, stability, or activity."
The symbiont in this case alters the levels of RNA by mechanisms other than transcription regulation, which is what the term GO:0052026 modulation by symbiont of host transcription refers to.
The paper further says "Further molecular analysis revealed that AvrPto-mediated reduction in miRNA accumulation may occur, at least in part, at the post-transcriptional level because AvrPto did not alter pri-miRNA transcript levels (Fig. S7A). Accordingly, conditional AvrPto expression in stable transgenic lines stabilizes miR393 precursors and concomitantly decreases accumulation of mature miR393, with no or little effects on pri-miR393 transcript levels (Fig. 3B/C, S7B). Therefore, AvrPto possibly interferes with processing of some miRNA precursors, a phenomenon also observed during Pto DC3000 infection (Fig. 3C, S8)."

In conclusion, the 2 terms seem to have different intended and non-overlapping usages, so it seems fine to have them both. I hope this makes sense, please let me know if I can be of further help.

Cheers,
Melanie

**Comment by LornaMin** [comment]
Thanks for that explanation.
